lib/facebook.ex

```diff
@@ -227,6 +227,23 @@ defmodule Facebook do
       |> GraphAPI.post("", [], params: params)
       |> ResponseFormatter.format_response
   end
```

Comment:
Can you call the new publish from this function now? Or rather, is it even needed now that the generic publish is here?

Reply:
Ah good point! I think it could be used here too. It can also safely be removed, what do you think? The only difference is in the type of first parameter which can anyway be a symbol indeed.

Reply:
@mweibel can you help in understanding why the build is failing please?